<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowFuelDeliveriesDialog</class>
 <widget class="QDialog" name="ShowFuelDeliveriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Livraisons de combustibles</string>
  </property>
  <property name="windowIcon">
   <iconset theme="clock">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Fuels">
     <property name="title">
      <string>Livraisons de combustibles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vous pouvez modifier les caractéristiques des livraisons enregistrées à partir du tableau ci-dessous en double-cliquant sur les cases concernées. La dernière colonne vous permet de supprimer des livraisons si nécessaire.&lt;/p&gt;&lt;p&gt;Les modifications seront automatiquement enregistrées.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableFuels">
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>UUID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Combustible</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantité livrée</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unité</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pouvoir Calorifique</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Humidité</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Énergie livrée</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Facture</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prix au MWh</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_NaturalGas">
     <property name="title">
      <string>Index du compteur de gaz naturel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidget_NaturalGas">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <row>
         <property name="text">
          <string>Date</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Index</string>
         </property>
        </row>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Electricity">
     <property name="title">
      <string>Index du compteur d'électricité</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTableWidget" name="tableWidget_Electricity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <row>
         <property name="text">
          <string>Date</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Index</string>
         </property>
        </row>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShowFuelDeliveriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShowFuelDeliveriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
